flixel/FlxG.hx

```diff
@@ -320,6 +321,11 @@ class FlxG
 	 */
 	public static var plugins(default, null):PluginFrontEnd;
 
+	/**
+	 * Used for detecting text input and dispatching commands on input text fields.
+	 */
+	public static var inputText(default, null):InputTextFrontEnd = new InputTextFrontEnd();
+
```

Comment:
Oh also, I recommend handling this, in the same manner as [FlxTweens and FlxTimer](https://github.com/HaxeFlixel/flixel/blob/ac488bf401880dbd5248b04f51a9c3492c8a39b8/flixel/system/frontEnds/PluginFrontEnd.hx#L138-L140):

Reply:
Where should the `globalManager` variable go though? The manager is intended to support more than just FlxInputText (e.g. an FlxBitmapInputText down the line), so putting it in that class doesn't seem like the best option.

Reply:
For now lets put it in FlxInputText, in the future when FlxBitmapInputText is made we could always have a static `globalManager` getter that references FlxInputText.globalManager. Alternatively, we might move this to a more centralized location, later